doc/bucket-class-crd.md

```diff
@@ -100,22 +102,17 @@ status:
 ```
 
 
-# Example
-
-Here are some examples of the cli/YAML usage and BucketClass CRs for the different bucket-class configurations:
-
-Single tier, single backing-store, placement Spread:
+# Examples
+Single tier, single backing store, Spread placement:
 ```shell
 noobaa -n noobaa bucketclass create placement-bucketclass bc --backingstores bs --placement Spread
```

Comment:
You changed the namespace to `app-namespace`

Reply:
I understand that you changed the namespace to `app-namespace` intentionally, I mean that this change needs also be in the shell commands: it is `-n nooba`a but should be changed to `-n app-namespace`. It is relevant for all the shell commands with -n flag.